tests/perf-test/SearchRouter.perf-test.tsx

```diff
@@ -117,6 +117,7 @@ function SearchAutocompleteInputWrapper() {
                 value={value}
                 onSearchQueryChange={(searchTerm) => setValue(searchTerm)}
                 isFullWidth={false}
+                substitutionMap={{}}
```

Comment:
NAB, but if you want you can use `CONST.EMPTY_OBJECT` here, just to get familiar with this const :D